LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -199,6 +204,8 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                             break;
                         }
 
+                        JoinRequestCounter.Add(1, new[] { KeyValuePair.Create(MetricRegistry.GatewayIdTagName, (object)stationEui) });
```

Comment:
Should we track the join request after the mic was successfully checked? At this stage could still be from another network?

Reply:
Addressed this in 07808a34fe97b4981c5d12ca4eeeb025a077d4af